clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/controllers/CloudFoundryLoadBalancerController.groovy

```diff
@@ -17,83 +17,78 @@
 package com.netflix.spinnaker.clouddriver.cf.controllers
 
 import com.fasterxml.jackson.annotation.JsonProperty
-import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider
-import com.netflix.spinnaker.clouddriver.cf.model.CloudFoundryLoadBalancer
-import com.netflix.spinnaker.clouddriver.cf.model.CloudFoundryResourceRetriever
+import com.netflix.spinnaker.cats.cache.Cache
+import com.netflix.spinnaker.cats.cache.CacheData
+import com.netflix.spinnaker.cats.cache.RelationshipCacheFilter
+import com.netflix.spinnaker.clouddriver.cf.cache.Keys
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestMethod
 import org.springframework.web.bind.annotation.RestController
 
+import static com.netflix.spinnaker.clouddriver.cf.cache.Keys.Namespace.LOAD_BALANCERS
+
 @RestController
 @RequestMapping("/cf/loadBalancers")
 class CloudFoundryLoadBalancerController {
 
-  @Autowired
-  AccountCredentialsProvider accountCredentialsProvider
+  private final Cache cacheView
 
   @Autowired
-  CloudFoundryResourceRetriever cfResourceRetriever
+  CloudFoundryLoadBalancerController(Cache cacheView) {
+    this.cacheView = cacheView
+  }
 
   @RequestMapping(method = RequestMethod.GET)
   List<CloudFoundryLoadBalancerAccount> list() {
-    getSummaryForLoadBalancers(cfResourceRetriever.loadBalancersByAccount).values() as List
+    Collection<String> loadBalancers = cacheView.getIdentifiers(LOAD_BALANCERS.ns)
+    getSummaryForLoadBalancers(loadBalancers).values() as List
   }
 
   @RequestMapping(value = "/{name:.+}", method = RequestMethod.GET)
   CloudFoundryLoadBalancerSummary get(@PathVariable String name) {
-    getSummaryForLoadBalancers(cfResourceRetriever.loadBalancersByAccount).get(name)
+    def searchKey = Keys.getLoadBalancerKey(name, '*', '*')  + "*"
```

Comment:
my bad, the ordering is correct, but I don't understand why the +  '*' or the findAll afterword is necessary - the globbing should accomplish a match on name by default